content/k8s/containers-with-docker/certbot/_index.md

```diff
@@ -6,6 +6,7 @@ from_repo: k8s/manual-app-deployment/project-overview
 prerequisites:
   hard:
   - k8s/containers-with-docker/containers-overview
+  - k8s/manual-app-deployment/project-overview
```

Comment:
hey there! 

Why is this a prereq for both the whole chapter? Is this for better Tilde integration?

Reply:
That is the source of the continue repo, so it needs to be a pre-req of every project that uses that repo